tox.ini

```diff
@@ -390,7 +390,7 @@ setenv =
   ; override CORE_REPO_SHA via env variable when testing other branches/commits than main
   ; i.e: CORE_REPO_SHA=dde62cebffe519c35875af6d06fae053b3be65ec tox -e <env to test>
   CORE_REPO_SHA={env:CORE_REPO_SHA:main}
-  CORE_REPO=git+https://github.com/open-telemetry/opentelemetry-python.git@{env:CORE_REPO_SHA}
+  CORE_REPO=git+https://github.com/xrmx/opentelemetry-python.git@async-asgi-test-base
```

Comment:
We need this to have tests green until https://github.com/open-telemetry/opentelemetry-python/pull/4106 get merged

Reply:
Done